### PR TITLE
Add cloudbuild yaml to the project slug

### PR DESCRIPTION
**Pull request by imjohsep**

### CODE REVIEW

**Comment by imjohsep** [comment]
@ray-ray Since this is a open source project, do you think it would be better to add a toggle to whether the google cloudbuild yaml is included? Can we do that with cookie cutter?

**Comment by imjohsep** [comment]
@ray-ray I've pushed up the cloudbuild removal as well as some changes to the prompt. If you don't like the prompt changes I'll revery them.

File: README.md [hunk 1]
```diff
@@ -16,11 +16,13 @@ Variable | Description
 ---------|------------
 project_name | Human readable name of your microservice
 project_slug | Machine-friendly version of project_name (directories/code/etc. will use this)
+project_upper | Upper case version of the project slug
 app_name | Importable name of your app (should differ from project_slug)
 dc_port | Port that will appear in the docker-compose.yml for the local environment
-use_sqlalchemy | True/False whether to include SQLAlchemy modules/code for MariaDB
+use_sqlalchemy | y/n whether to include SQLAlchemy modules/code for MariaDB
```
Comment (ray-ray) [inline README.md]: Why switch to y/n? Just want less typing?

Comment (imjohsep) [reply README.md]: Yeah, and if you type in true/false vs True/False you get an error in python.

**Pull request merged**